### PR TITLE
feat: Panel tab style

### PR DESCRIPTION
**Pull request by ianxinnew**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

### What is the new behavior?
image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **Style**
  - Updated tab styles and adjusted padding for improved layout.

- **Refactor**
  - Renamed constants for better clarity and updated import paths to enhance module structure.

### CODE REVIEW

File: packages/toolbars/breadcrumb/src/Main.vue [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default {
     TinyButton: Button
   },
   setup() {
-    const CONTENTS = {
+    const PLUGINSID = {
```
Comment (wenmine) [inline packages/toolbars/breadcrumb/src/Main.vue]: PLUGINSID -> PLUGINS_ID

File: packages/toolbars/breadcrumb/src/Main.vue
Comment (chilingling) [inline packages/toolbars/breadcrumb/src/Main.vue:1]: 26行 不可以以相对路径的方式进行 import

File: packages/layout/src/DesignSettings.vue [hunk 2]
```diff
@@ -55,14 +55,21 @@ export default {
...
     .tiny-tabs__nav-scroll {
-      text-align: center;
+      margin-left: 12px;
       .tiny-tabs__nav {
         display: inline-flex;
         justify-content: center;
         float: none;
       }
+      .tiny-tabs__active-bar {
+        width: 40px !important;
```
Comment (chilingling) [inline packages/layout/src/DesignSettings.vue:66]: 为什么需要 `!important` ？

Comment (ianxinnew) [reply packages/layout/src/DesignSettings.vue:66]: tiny-tabs__active-bar有行内样式style，需要提高层级

**Review by hexqi — APPROVED** [review]

**Pull request merged**